app/components/post_attachment_opengraph/post_attachment_opengraph.js

```diff
@@ -216,7 +216,6 @@ export default class PostAttachmentOpenGraph extends PureComponent {
                     style={{width, height}}
                 >
                     <Image
-                        ref='image'
```

Comment:
This ref is not used.

Reply:
are you sure??????!!!!?? I thought that one is used for the transition when you tap on the image to go to the image preview

Reply:
Yes, the ref being used is `item` which is the container of this image.

Tapping of the image preview works fine.

https://github.com/mattermost/mattermost-mobile/blob/master/app/components/post_attachment_opengraph/post_attachment_opengraph.js#L169